CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ CHANGES IN VERSION 1.18.1 (FROM 1.18.0)
 
 Appium 1.18.1 is a patch release
 
+#### Android General
+* Fix to kill forwarded port socket connection [appium-chromedriver#183](https://github.com/appium/appium-chromedriver/pull/183)
```

Comment:
I'd rather say: Fix to properly clean up forwarded ports if parallel sessions are running